Sofa/framework/Simulation/Core/src/sofa/simulation/mechanicalvisitor/MechanicalComputeForceVisitor.h

```diff
@@ -34,11 +34,10 @@ class SOFA_SIMULATION_CORE_API MechanicalComputeForceVisitor : public Mechanical
 public:
     sofa::core::MultiVecDerivId res;
     bool accumulate; ///< Accumulate everything back to the DOFs through the mappings
-    bool neglectingCompliance; /// neglect Compliance?
 
     MechanicalComputeForceVisitor(const sofa::core::MechanicalParams* mparams,
-                                  sofa::core::MultiVecDerivId res, bool accumulate = true,  bool neglectingCompliance = true )
-            : MechanicalVisitor(mparams) , res(res), accumulate(accumulate), neglectingCompliance(neglectingCompliance)
+                                  sofa::core::MultiVecDerivId res, bool accumulate = true )
```

Comment:
I am also against. The visitor still exists. If the compiler triggers an error, it will not be so complex to figure out what's going on. But I add the label breaking to this PR